blosc/blosc2.c

```diff
@@ -1812,9 +1813,9 @@ int blosc_compress_context(blosc2_context* context) {
 }
 
 
-/* The public routine for compression with context. */
-int blosc2_compress_ctx(blosc2_context* context, size_t nbytes,
-                        const void* src, void* dest, size_t destsize) {
+/* The public secure routine for compression with context. */
+int blosc2_compress_ctx(blosc2_context* context, const void* src, size_t srcsize,
+                        void* dest, size_t destsize) {
```

Comment:
To be consistent with the above, `srcsize` and `destsize` should be changed to `int32_t`.  Could you do it here?

Reply:
Yes I think fixed width is the best for an api.